.changeset/nice-mirrors-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@crowdstrike/glide-core': patch
+---
+
+`preventDefault()` can now be called on Popover's target to prevent Popover from opening or closing when the target is clicked.
```

Comment:
```suggestion
`preventDefault()` can now be called on Popover's target "click" event to prevent Popover from opening or closing when the target is clicked.
```